cg/exc.py

```diff
@@ -247,3 +247,10 @@ class LoqusdbDeleteCaseError(LoqusdbError):
 
 class LoqusdbDuplicateRecordError(LoqusdbError):
     """Exception related to duplicate records in Loqusdb."""
+
+class PdcNoFilesMatchingSearchError(CgError):
+    """Exception raised when PDC API returns no files matching the search criteria."""
+
+    def __init__(self, message: str = ""):
+        super().__init__(message)
+        self.exit_code = 8
```

Comment:
Could you add a named constant representing the error so that one does not have to know what exit code "8" stands for

Reply:
Is this not already kind of a constant except that it is an Error with an attribute instead of a set constant?

Reply:
I mean the actual “8”, should be replaced. 8 seems to mean something like: Process Exit Code 8 translates to Tivoli Storage Manager return code 8. This means the backup completed, but witnessed one or more warnings. it would be nice to capture this in a named constant that tells you something about exit code 8. It will immediately put this context when you read the code - 8 does not tell me much and even less in 3 months 😀

Reply:
Well - the error itself is raised and appropriate message given. Somewhere in the code the constant will have to be a 8. I just decided to here place it inside the error code instead of another loose constant as I feel that the exit code 8 and what it means is specific to this particular error. But I will se this as a constant and import it :)